### PR TITLE
Add license headers for gradle files

### PR DESCRIPTION
**Pull request by pahans**
## Purpose
Add license headers for gradle files

### CODE REVIEW

File: bvm/ballerina-core/build.gradle [hunk 1]
```diff
@@ -2,6 +2,23 @@ plugins {
     id "com.github.harbby.gradle.serviceloader" version "1.1.5"
 }
 
+/*
```
Comment (Shan1024) [inline bvm/ballerina-core/build.gradle]: There is a plugin closure at the top of the file.

File: bvm/ballerina-streaming/siddhi-query-compiler/build.gradle [hunk 2]
```diff
@@ -2,6 +2,23 @@ plugins {
     id 'antlr'
 }
 
+/*
```
Comment (Shan1024) [inline bvm/ballerina-streaming/siddhi-query-compiler/build.gradle]: There is a plugin closure at the top of the file.

File: composer/build.gradle [hunk 3]
```diff
@@ -1,4 +1,21 @@
 import org.apache.tools.ant.taskdefs.condition.Os
+/*
```
Comment (Shan1024) [inline composer/build.gradle]: There is an import at the top of the file.

File: distribution/zip/ballerina-tools/build.gradle [hunk 4]
```diff
@@ -2,6 +2,23 @@ plugins {
     id 'base'
 }
 
+/*
```
Comment (Shan1024) [inline distribution/zip/ballerina-tools/build.gradle]: There is a plugin closure at the top of the file.

File: distribution/zip/ballerina/build.gradle [hunk 5]
```diff
@@ -2,6 +2,23 @@ plugins {
     id 'base'
 }
 
+/*
```
Comment (Shan1024) [inline distribution/zip/ballerina/build.gradle]: There is a plugin closure at the top of the file.

File: gradle/javaProject.gradle [hunk 6]
```diff
@@ -1,8 +1,93 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply plugin: 'java'
+/*
```
Comment (Shan1024) [inline gradle/javaProject.gradle]: Redundant license header.

File: gradle/javaProject.gradle [hunk 7]
```diff
@@ -1,8 +1,93 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply plugin: 'checkstyle'
+/*
```
Comment (Shan1024) [inline gradle/javaProject.gradle]: Redundant licence headers.

File: gradle/javaProject.gradle [hunk 8]
```diff
@@ -1,8 +1,93 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply plugin: "com.github.spotbugs"
+/*
```
Comment (Shan1024) [inline gradle/javaProject.gradle]: Redundant licence headers.

File: gradle/javaProject.gradle [hunk 9]
```diff
@@ -1,8 +1,93 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply plugin: 'jacoco'
 
+/*
```
Comment (Shan1024) [inline gradle/javaProject.gradle]: Redundant licence headers.

File: gradle/jsProject.gradle [hunk 10]
```diff
@@ -1,4 +1,21 @@
 import org.apache.tools.ant.taskdefs.condition.Os
+/*
```
Comment (Shan1024) [inline gradle/jsProject.gradle]: There is an import at the top of the file.

File: misc/docerina/build.gradle [hunk 11]
```diff
@@ -1,4 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply from: "$rootDir/gradle/javaProject.gradle"
+/*
```
Comment (Shan1024) [inline misc/docerina/build.gradle]: Redundant licence headers.

File: misc/openapi-ballerina/modules/ballerina-client-generator/build.gradle [hunk 12]
```diff
@@ -1,4 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply from: "$rootDir/gradle/javaProject.gradle"
+/*
```
Comment (Shan1024) [inline misc/openapi-ballerina/modules/ballerina-client-generator/build.gradle]: Redundant licence headers.

File: misc/openapi-ballerina/modules/openapi-to-ballerina-generator/build.gradle [hunk 13]
```diff
@@ -1,4 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply from: "$rootDir/gradle/javaProject.gradle"
+/*
```
Comment (Shan1024) [inline misc/openapi-ballerina/modules/openapi-to-ballerina-generator/build.gradle]: Redundant licence headers.

File: misc/strip-bouncycastle/build.gradle [hunk 14]
```diff
@@ -1,4 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply from: "$rootDir/gradle/javaProject.gradle"
+/*
```
Comment (Shan1024) [inline misc/strip-bouncycastle/build.gradle]: Redundant licence headers.

File: stdlib/messaging/rabbitmq/build.gradle [hunk 15]
```diff
@@ -16,6 +16,23 @@
  * under the License.
  */
 
+/*
```
Comment (Shan1024) [inline stdlib/messaging/rabbitmq/build.gradle]: Redundant licence headers.

File: stdlib/nats/build.gradle [hunk 16]
```diff
@@ -16,6 +16,23 @@
  * under the License.
  */
 
+/*
```
Comment (Shan1024) [inline stdlib/nats/build.gradle]: Redundant licence headers.

File: tests/ballerina-test-utils/build.gradle [hunk 17]
```diff
@@ -1,4 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply from: "$rootDir/gradle/javaProject.gradle"
+/*
```
Comment (Shan1024) [inline tests/ballerina-test-utils/build.gradle]: Redundant licence headers.

File: tests/composer-integration-test/build.gradle [hunk 18]
```diff
@@ -1,4 +1,21 @@
 import org.apache.tools.ant.taskdefs.condition.Os
+/*
```
Comment (Shan1024) [inline tests/composer-integration-test/build.gradle]: There is an import at the top of the file.

File: tests/observability-test-utils/build.gradle [hunk 19]
```diff
@@ -1,4 +1,38 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+ 
 apply from: "$rootDir/gradle/balNativeLibProject.gradle"
+/*
```
Comment (Shan1024) [inline tests/observability-test-utils/build.gradle]: Redundant licence headers.

File: tool-plugins/vscode/build.gradle [hunk 20]
```diff
@@ -1,4 +1,21 @@
 import org.apache.tools.ant.taskdefs.condition.Os
+/*
```
Comment (Shan1024) [inline tool-plugins/vscode/build.gradle]: There is an import at the top of the file.

**Comment by pahans** [comment]
@Shan1024 seems find & replace went wrong. Will fix and reopen

**Pull request merged**